task_2/andrii.lukianchykov/task_2/task_2/task_2/src/main/java/com/bookshop/bookdatabase/BookDataBase.java

```diff
@@ -0,0 +1,85 @@
+package com.bookshop.bookdatabase;
+
+import com.bookshop.book.Book;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.lang.System.out;
+
+public class BookDataBase {
+
+    private Book book;
+    private static List<Book> listOfBooks = new ArrayList<>();
+
+    public BookDataBase() {}
```

Comment:
If you are not using a default constructor you don't have to define it.

Reply:
Got it